bluemira/base/reactor.py

```diff
@@ -204,3 +217,29 @@ def show_cad(
             self._construct_xyz_cad(comp_copy, with_components, n_sectors)
 
         ComponentDisplayer().show_cad(comp_copy, **kwargs)
+
+    def plot(self, *dims: str, with_components: Optional[List[ComponentManager]] = None):
+        """
+        Plot the reactor.
+
+        Parameters
+        ----------
+        *dims:
+            The dimension(s) of the reactor to show, 'xz' and/or 'xy'.
+            (default: 'xz')
+        with_components:
+            The components to construct when displaying CAD for xyz.
+            Defaults to None, which means show "all" components.
+        """
+        # give dims_to_show a default value
+        dims_to_show = ("xz",) if len(dims) == 0 else dims
+
+        for dim in dims_to_show:
+            if dim not in _PLOT_DIMS:
+                raise ReactorError(
+                    f"Invalid plotting dimension '{dim}'."
+                    f"Must be one of {str(_PLOT_DIMS)}"
+                )
+        for i, dim in enumerate(dims_to_show):
+            comp_copy = self._filter_copy_comps([dim], with_components)
+            ComponentPlotter().plot_2d(comp_copy, show=i == len(dims_to_show) - 1)
```

Comment:
When plotting xy, the labels for the plot are incorrect

![image](https://user-images.githubusercontent.com/3520445/229101284-ba1c6cc3-4c6e-4776-8871-8dc287f15660.png)

Reply:
Yep good catch I'm on it